### PR TITLE
Fetch polyfill

### PR DESCRIPTION
**Pull request by zzarcon**

### CODE REVIEW

**Comment by rpunkfu** [comment]
After my push gh-emoji works on Safari. @zzarcon after review please pull my commit and bump the version of package so I can update gh-pages :)

After this we need to separate src into modules and export fetchPolyfill into helpers or smth.

screen

**Comment by zzarcon** [comment]
@oskarcieslik take a last look please :)

**Comment by rpunkfu** [comment]
Yup, merging :)

File: src/index.js [hunk 1]
```diff
@@ -10,6 +10,24 @@ type ParseOptions = {
   classNames?: string,
 };
 
+if (!window.fetch) {
+  function fetch(text: string): Promise < Object > {
```
Comment (ryyppy) [inline src/index.js]: Please no spaces between `<` / `>` ... `Promise<Object>`

**Pull request merged**